web/Services/AdminService.cs

```diff
@@ -47,6 +48,14 @@ public AdminService(BricksAndHeartsDbContext dbContext, ILogger<AdminService> lo
         UserDbModel userFromId = _dbContext.Users.SingleOrDefault(u => u.Id == userId)!;
         return userFromId;
     }
+    
+    public AdminCountModel CountAdmins()
+    {
+        AdminCountModel adminCounts = new AdminCountModel();
+        adminCounts.CurrentAdmins = _dbContext.Users.Count(u => u.IsAdmin == true);
+        adminCounts.PendingAdmins = _dbContext.Users.Count(u => u.HasRequestedAdmin == true && u.IsAdmin == false);
+        return adminCounts;

```

Comment:
this could just be 
```suggestion
        return new AdminCountModel
        {
            CurrentAdmins = _dbContext.Users.Count(u => u.IsAdmin == true),
            PendingAdmins = _dbContext.Users.Count(u => u.HasRequestedAdmin == true && u.IsAdmin == false)
        };

```